Refactors merge conflict resolution

Lindayezou opened the pull request:
Shows styled view and can filter lists properly

 Changes to be committed:
    modified:   home/home.html

Commit:
Refactors merge conflict resolution

Shows styled view and can filter lists properly

 Changes to be committed:
	modified:   home/home.html

hiteshlala commented:
tested, this has some bugs that can easily be fixed before merge - look at in line comments.

Commit:
Changes color of 'choose me' on the starter pokemon page

 Changes to be committed:
	modified:   style.css

Commit:
Adds pokeball to sign in page

Deletes commented out text in home.html

 Changes to be committed:
	modified:   auth/signin.html
	modified:   home/home.html
	modified:   style.css

Commit:
Ensures all pages conform to same button style

 Changes to be committed:
	modified:   board/board.html
	modified:   capturePokemon/capturePokemon.html
	modified:   city/city.html
	modified:   event/event.html
	modified:   home/home.html
	modified:   lobby/lobby.html

Commit:
Adds Pokeball file
 Changes to be committed:
	new file:   assets/pokeball.png

hiteshlala commented on public/home/home.html:
```diff
@@ -1,24 +1,54 @@
+      <div class='newGameContent col-lg-5'>    
+        <td>
+        <form name="myform">
+          <input class="amount" type="text" size="12" ng-model="newGameName" placeholder="Name Required!" ng-keypress="hitEnter($event, input)"></input>
```
remove input from hitEnter function it is not used - look at line 6 of previous code

hiteshlala commented on public/home/home.html:
```diff
@@ -1,24 +1,54 @@
+          <h2 class='panel-heading'><b>{{playerName}}'s Games</b></h2>
+          <div class='gameContent' style="height: 300px">
+            <li ng-repeat = "game in myGames track by $index">
+              <a href='#/lobby' ng-click='joinLobby(game.gameId)'>{{game.gameId}}: {{game.gameName}} </a>
```
This needs to be changed.  The redirect does not happen from the client side any more

Look at line 15 of old code

hiteshlala commented on public/auth/signin.html:
```diff
@@ -1,8 +1,9 @@
-  <div class='login facebook'>
-	 <a href="/signin/facebook" target="_self"><img src="http://i.stack.imgur.com/Vk9SO.png"></a> 
+  <div class='login facebook col-lg-12'>
+  <image class="pokeball" src="assets/pokeball.png" x="0" y="0"  width="500px" height="500px" />
```
reduce the size of ball a bit to maybe 400px

hiteshlala commented on public/home/home.html:
```diff
@@ -1,24 +1,54 @@
+            <h2 class='panel-heading' ><b>All Available Games</b></h2>
+            <div class='gameContent' style="height: 300px">
+              <li ng-repeat = "game in games track by $index" ng-hide="game.gameStarted">
+                <a href='#/lobby' ng-click='joinLobby(game.gameId)'>{{game.gameId}}: {{game.gameName}} </a>
```
same here, redirect is on server side.  Look at line 20 of old code

Commit:
Made changes for pull request comments

Also minor style edits to signin page

 Changes to be committed:
	modified:   auth/signin.html
	modified:   home/home.html
	modified:   index.html
	modified:   style.css

hiteshlala commented:
Reviewed.
